grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlers.java

```diff
@@ -114,7 +114,7 @@ static UnframedGrpcErrorHandler of(
      * @param statusMappingFunction The function which maps the {@link Throwable} or gRPC {@link Status} code
      *                              to an {@link HttpStatus} code.
      */
-    static UnframedGrpcErrorHandler ofJson(
+    static JsonUnframedGrpcErrorHandler ofJson(
```

Comment:
Should we move this logic to `JsonUnframedGrpcErrorHandler` and make `UnframedGrpcErrorHandlers` small?